src/ui/public/vislib/styles/_tooltip.less

```diff
@@ -1,10 +1,12 @@
 @import (reference) "~ui/styles/variables";
 
+@semi-bold: 600;
```

Comment:
Put this in variables.less, it seems reusable. Though do we really need another font-weight?